frontend/components/product-list/sections/FilterableProductsSection/FacetsAccordion.tsx

```diff
@@ -89,10 +89,12 @@ type FacetAccordionItemProps = AccordionItemProps & {
 export const FacetAccordionItem = forwardRef<FacetAccordionItemProps, 'div'>(
    ({ attribute, refinedCount, productList, isExpanded, ...props }, ref) => {
       const { items } = useFilteredRefinementList({ attribute });
+      const { isEmptyFacetState } = useFilteredRefinementList({ attribute });
       const { hits } = useHits();
       const isProductListEmpty = hits.length === 0;
       const hasApplicableRefinements = items.length > 0;
-      const isDisabled = isProductListEmpty || !hasApplicableRefinements;
+      const isDisabled =
+         isProductListEmpty || !hasApplicableRefinements || isEmptyFacetState;
 
       const formattedFacetName = formatFacetName(attribute);
       const isHidden = !hasApplicableRefinements && !isProductListEmpty;
```

Comment:
Would this achieve a similar effect?

```suggestion
      const isDisabled = isProductListEmpty || !hasApplicableRefinements;

      const formattedFacetName = formatFacetName(attribute);
      const isHidden = !isProductListEmpty;
```

Reply:
Nope this will just hide it whenever the product list is non empty, silly me